core/src/main/java/com/homeaway/streamplatform/streamregistry/model/Stream.java

```diff
@@ -121,11 +124,13 @@
     /**
      * Number of Partitions
      */
+    @ApiModelProperty(example = "1")
     int partitions;
 
     /**
      * Replication Factor
```

Comment:
Will the replicationFactor reflect as 3 in the Swagger ?

If so, then it may fail, because, usually OSS testing in laptop will have only 1 Broker clusters

Reply:
@prabhakar1983 as discussed setting replicationFactor as 1 by default would be a wrong standard in production. In local they might have to update the replicationFactor to 1

Reply:
agreed that we should opt for production, and maybe provide a readme or documentation for local developer values.

Reply:
👍